<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cfgsettingdyn</class>
 <widget class="QWidget" name="cfgsettingdyn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Response</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="Con_Dyn_Module" name="dynGraph" native="true">
        <property name="minimumSize">
         <size>
          <width>306</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>306</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Applied</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="VolumeWidget" name="appliedGain" native="true">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtGain">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Setting</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <property name="spacing">
       <number>9</number>
      </property>
      <item row="12" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Bypass</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="lblDynRadio">
        <property name="text">
         <string>Ratio
(n:1)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="ConfigGainWidget" name="_gainTarget" native="true">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="lblTimeAttack">
        <property name="text">
         <string>Attack
Time(ms)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="lblTimeRelease">
        <property name="text">
         <string>Release
Time(ms)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="ConfigGainWidget" name="_timeRelease" native="true">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="lblGainTarget">
        <property name="text">
         <string>Target
(dB)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="ConfigGainWidget" name="_timeAttack" native="true">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="10" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="ConfigGainWidget" name="_dynRatio" native="true">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="ConfigGainWidget" name="_gainThreshold" native="true">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QPushButton" name="pb_bypass">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="lblGainThreshold">
        <property name="text">
         <string>Threshold
(dB)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="11" column="0">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigGainWidget</class>
   <extends>QWidget</extends>
   <header>customWidget/configgainwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VolumeWidget</class>
   <extends>QWidget</extends>
   <header>customWidget/volumewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Con_Dyn_Module</class>
   <extends>QWidget</extends>
   <header>customWidget/con_dyn_module.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
